packages/hoprd/crates/hoprd-api/src/lib.rs

```diff
@@ -2,6 +2,7 @@ pub mod config;
 
 use std::{sync::Arc, collections::HashMap};
 use std::error::Error;
+use std::{collections::HashMap, sync::Arc};
```

Comment:
Duplicate import of `std::sync::Arc`.

```suggestion
use std::collections::HashMap;
```

Reply:
This should not have even compiled with out settings.